### PR TITLE
Fix #6893 - Added new tab open button for landscape UI

### LINKED ISSUE #6893: New Tab Button UI (landscape)
### Vision

As a user viewing a webpage, I want a button to start a brand new search in a new tab so that I can quick start another search while keeping my current tab open for later

### Acceptance Criteria

* A new tab button has been added to nav bar in landscape mode [spec](https://docs.google.com/document/d/1n0UDbk2UexFB360uoGLwIvjkaMaQ_ZyXmmLTigBfJ3g/edit#heading=h.p1cuc1sh5ua) (landscape only)
  * New tab button on toolbar in landscape mode
  * Button opens to a new tab with the keyboard up, ready to type in URL bar
* Ensure a11y standards are met

### Not included

* Solution for portrait mode will be a separate ticket
* Telemetry in #6886

### PR DESCRIPTION
**Pull request by nbhasin2**
The ➕ button should only be shown in landscape mode beside URL Bar

image

### CODE REVIEW

File: Client/Frontend/Browser/URLBarView.swift [hunk 1]
```diff
@@ -152,8 +152,8 @@ class URLBarView: UIView {
 
     var appMenuButton = ToolbarButton()
     var libraryButton = ToolbarButton()
+    var addNewTabButton = ToolbarButton()
 
-    var bookmarkButton = ToolbarButton()
```
Comment (nbhasin2) [inline Client/Frontend/Browser/URLBarView.swift:156]: This button wasn't being used anywhere in URLBar hence removed it,.

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 2]
```diff
@@ -236,7 +236,7 @@ class BrowserViewController: UIViewController {
 
         urlBar.topTabsIsShowing = showTopTabs
         urlBar.setShowToolbar(!showToolbar)
-
+        toolbar?.addNewTabButton.isHidden = showToolbar
```
Comment (nbhasin2) [inline Client/Frontend/Browser/BrowserViewController.swift:239]: This part is necessary if user opens in a landscape mode

File: Client/Frontend/Browser/BrowserViewController/BrowserViewController+TabToolbarDelegate.swift [hunk 3]
```diff
@@ -54,6 +54,12 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
             showLibrary()
         }
     }
+    
+    func tabToolbarDidPresssAddNewTab(_ tabToolbar: TabToolbarProtocol, button: UIButton) {
```
Comment (nbhasin2) [inline Client/Frontend/Browser/BrowserViewController/BrowserViewController+TabToolbarDelegate.swift:58]: Added new method to handle when ➕ is tapped we open a new tab and focus on the URL Bar so user can easily search.

Comment (garvankeeley) [reply Client/Frontend/Browser/BrowserViewController/BrowserViewController+TabToolbarDelegate.swift:58]: To many s in `Presss`

File: Client/Frontend/Browser/TabToolbar.swift [hunk 4]
```diff
@@ -122,6 +123,11 @@ open class TabToolbarHelper: NSObject {
         let longPressGestureTabsButton = UILongPressGestureRecognizer(target: self, action: #selector(didLongPressTabs))
         toolbar.tabsButton.addGestureRecognizer(longPressGestureTabsButton)
 
+        toolbar.addNewTabButton.setImage(UIImage.templateImageNamed("menu-NewTab"), for: .normal)
+        toolbar.addNewTabButton.accessibilityLabel = "Some Label"
```
Comment (garvankeeley) [inline Client/Frontend/Browser/TabToolbar.swift:127]: missed filling this in

**Comment by nbhasin2** [comment]
Going to do some testing for accessibility before merging this in.

**Pull request merged**